test/utils/test_cfg_builder.py

```diff
@@ -0,0 +1,38 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2020 Aibolit
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+from unittest import TestCase
+from pathlib import Path
+
+from aibolit.utils.java_package import JavaPackage
+from aibolit.utils.java_class import JavaClass
+from aibolit.utils.java_class_method import JavaClassMethod
+
+
+class CFGBuilderTestCase(TestCase):
+
+    def test_cfg_of_method(self):
+        java_package = JavaPackage(Path(__file__).parent.absolute() / "SimpleClass.java")
+        fst: JavaClass = [c for c in java_package.java_classes][0]
```

Comment:
We could omit type declarations. If types are not deduced automatically now, because of `@property  # type: ignore`, it will probably work after we would start using `@cached_property`. It seems to be supported by mypy.